pg_cron.spec

```
@@ -56,6 +56,8 @@ PATH=%{pginstdir}/bin:$PATH
 %endif
 
 %changelog
+* Mon Mar 29 2021 - Marco Slot <marco@citusdata.com> 1.3.1-1
```

Looking into other entries in this spec file, should we do ?

```suggestion
* Mon Mar 29 2021 - Marco Slot <marco@citusdata.com> 1.3.1.citus-1
```

---

In previous build there was an error about this issue so I removed citus and fixed. Warning is as below
https://travis-ci.com/github/citusdata/packaging/jobs/494490949#L644